util/src/test/scala/org/alephium/util/AVectorSpec.scala

```diff
@@ -385,6 +415,12 @@ abstract class AVectorSpec[@sp A: ClassTag](implicit ab: Arbitrary[A], cmp: Orde
       vc.indices.step is 1
     }
   }
+
+  it should "toIterable" in new Fixture {
+    forAll(vectorGen) { vc =>
+      vc.toIterable.toSeq is vc.toArray.toIterable.toSeq
```

Comment:
can't you directly compare the `Iterable` rather doing the `toSeq`?

Reply:
Unfortunately not. This might be a bug of Scala